### PR TITLE
add here_now to presence API

### PR DESCRIPTION
**Pull request by Xavrax**

### CODE REVIEW

File: examples/presence_state.rs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    //        .execute()
+    //        .await?;
+    //
+    //    println!("All channels state: {:?}", states);
+    //
+    //    states.iter().for_each(|channel| {
+    //        println!("Channel: {}", channel.channel);
+    //        println!("State: {:?}", channel.state);
+    //    });
+    //
```
Comment (parfeon) [inline examples/presence_state.rs:41]: Is it a placeholder for changes prepared in other branches?

File: examples/presence_state_blocking.rs [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    //        .user_id("user_id")
+    //        .execute_blocking()?;
+    //
+    //    println!("All channels state: {:?}", states);
+    //
+    //    states.iter().for_each(|channel| {
+    //        println!("Channel: {}", channel.channel);
+    //        println!("State: {:?}", channel.state);
+    //    });
+    //
```
Comment (parfeon) [inline examples/presence_state_blocking.rs:38]: Is it a placeholder for changes prepared in other branches?

File: examples/where_now_blocking.rs [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        .build()?;
+
+    println!("running!");
+
+    //    let where_user = client
+    //        .where_now()
+    //        .user_id("user_id")
+    //        .execute_blocking()?;
+    //
+    //    println!("All channels data: {:?}", where_user);
```
Comment (parfeon) [inline examples/where_now_blocking.rs:24]: Is it a placeholder for changes prepared in other branches?

File: examples/where_now.rs [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+    println!("running!");
+
+    //    let where_user = client
+    //        .where_now()
+    //        .user_id("user_id")
+    //        .execute()
+    //        .await?;
+    //
+    //    println!("All channels data: {:?}", where_user);
```
Comment (parfeon) [inline examples/where_now.rs:26]: Is it a placeholder for changes prepared in other branches?

File: src/dx/presence/builders/here_now.rs [hunk 5]
```diff
@@ -0,0 +1,187 @@
...
+            .then(|| query.insert("state".into(), "1".into()));
+
+        (!self.include_user_id).then(|| {
+            query.insert("disable_uuids".into(), "1".into());
+        });
+
+        query.insert(
+            "uuid".into(),
+            self.pubnub_client.config.user_id.deref().clone(),
+        );
```
Comment (parfeon) [inline src/dx/presence/builders/here_now.rs:125]: This one did not require because it is included by middleware with required encoding.

**Review by parfeon — CHANGES_REQUESTED** [review]
There is only one actual change request.

File: examples/presence_state.rs (hunk 1 above)
Comment (Xavrax) [reply examples/presence_state.rs:41]: Yup. Initially I wanted to make one PR with all presence methods but I noticed that this PR will be quite a big.

File: examples/presence_state_blocking.rs (hunk 2 above)
Comment (Xavrax) [reply examples/presence_state_blocking.rs:38]: Yup. Initially I wanted to make one PR with all presence methods but I noticed that this PR will be quite a big.

File: examples/where_now_blocking.rs (hunk 3 above)
Comment (Xavrax) [reply examples/where_now_blocking.rs:24]: Yup. Initially I wanted to make one PR with all presence methods but I noticed that this PR will be quite a big.

File: examples/where_now.rs (hunk 4 above)
Comment (Xavrax) [reply examples/where_now.rs:26]: Yup. Initially I wanted to make one PR with all presence methods but I noticed that this PR will be quite a big.

File: src/dx/presence/builders/here_now.rs (hunk 5 above)
Comment (Xavrax) [reply src/dx/presence/builders/here_now.rs]: fixed

**Review by parfeon — APPROVED** [review]

**Pull request merged**